setup.py

```diff
@@ -24,7 +24,7 @@ def read(*parts):
     url='https://github.com/python-babel/django-babel/',
     packages=find_packages(exclude=('tests',)),
     install_requires=[
-        'django>=1.4,<1.10',
+        'django>=1.4,<1.11',
         'babel>=1.3',
     ],
     classifiers=[
```

Comment:
comment for file below
You should delete tests/__init__.py 

> avoid “__init__.py” files in your test directories. This way your tests can run easily against an installed version of mypkg, independently from the installed package if it contains the tests or not.

http://doc.pytest.org/en/latest/goodpractices.html

:arrow_down: :arrow_down: :arrow_down: :arrow_down: :arrow_down: :arrow_down: